File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] arr, int t) {
        int n = arr.length;
        int out[] = new int[2];
        for (int i=0; i<n; i++){
            for (int j=0; j<n; j++){
                if (i!=j){
                 if (arr[i]+arr[j] == t){
                out[0] = i;
                out[1] = j;
                break;     
                }
            }
        }
           
    }
        return out;
    }
}

File: 0001-two-sum/0001-two-sum.js
/**
 * Function to find two numbers in an array that add up to a given target
 * using nested loops.
 * @param {number[]} nums - The input array of numbers.
 * @param {number} target - The target sum.
 * @return {number[]} - The indices of the two numbers that add up to the target.
 */

var twoSum = function(nums, target) {
    // Loop through each number in the array
    for (let i = 0; i < nums.length; i++) {
        // Loop through the numbers after the current one
        for (let j = i + 1; j < nums.length; j++) {
            // Check if the two numbers add up to the target
            if (nums[i] + nums[j] === target) {
                // Return their indices
                return [i, j];
            }
        }
    }
    // If no pair found, return an empty array
    return [];
};
